tracer/src/Datadog.Trace/Agent/MessagePack/MetaStructMessagePackHelper.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="MetaStructMessagePackHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Vendors.MessagePack.Formatters;
+
+#nullable enable
+
+namespace Datadog.Trace.Agent.MessagePack;
+
+internal static class MetaStructMessagePackHelper
+{
+    internal static byte[] ObjectToByteArray(object? value)
```

Comment:
If this is only used in tests, can we move it to one of the test projects?

Reply:
This class will be used by RASP, but until then, there is no problem about leaving it in test projects.

Reply:
Oh, if you plan on using it then it's fine to stay here. Sorry about that.

Reply:
No problem. I will leave it for now here until it's actually used :)